core/lib/dal/src/consensus_dal.rs

```diff
@@ -491,6 +552,45 @@ impl ConsensusDal<'_, '_> {
         // and doesn't have prunning enabled.
         Ok(attester::BatchNumber(0))
     }
+
+    pub async fn get_last_batch_committee_number(
```

Comment:
Can you add some docstrings to explain what this method is supposed to do? It sounds like "the committee number of the last batch" but there is no such thing. Is it the batch number of the last _executed_ batch, for which a committee is known? Isn't that available somewhere else?